<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Program Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetAllButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset All</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>327</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="updateIntervalLabel">
      <property name="text">
       <string>&lt;b&gt;Global process update interval:&lt;/b&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="updateIntervalSpinBox">
      <property name="suffix">
       <string> seconds</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="subUpdateIntervalLabel">
      <property name="text">
       <string>&lt;b&gt;Individual process update interval:&lt;/b&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="subUpdateIntervalSpinBox">
      <property name="suffix">
       <string> seconds</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="runningUpdateIntervalLabel">
      <property name="text">
       <string>&lt;b&gt;Running processes update interval:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="runningUpdateIntervalSpinBox">
      <property name="suffix">
       <string> seconds</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLabel" name="cpuUpdateIntervalLabel">
      <property name="text">
       <string>&lt;b&gt;Processor activity update interval:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="cpuUpdateIntervalSpinBox">
      <property name="suffix">
       <string> seconds</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="sortByLabel">
      <property name="text">
       <string>&lt;b&gt;Sort processes by:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="sortByComboBox">
      <item>
       <property name="text">
        <string>Process ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>State</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Priority</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QComboBox" name="sortByOrderComboBox">
      <item>
       <property name="text">
        <string>Ascending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Descending</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
